todo_app/data/create_item_demo.py

```diff
@@ -0,0 +1,19 @@
+import requests
```

Comment:
Similar to the `environment_test.py`, definitely recommend using these files as an intermediate step, but it's a good idea to clear them out after 🙂